north/src/main.rs

```diff
@@ -33,10 +33,27 @@ struct Opt {
     pub debug: bool,
 }
 
+async fn parse_config(path: &PathBuf) -> Result<Config> {
```

Comment:
We can get rid of this one line `fn` with the `run` function mentioned below.

Reply:
The config.debug is needed to configure the logging, that's why we have this double check for a possible exit(1).